README.md

```diff
@@ -1,10 +1,10 @@
 QupZilla Web Browser
 ----------------------------------------------------------------------------------------
 
-[![Build Status](https://travis-ci.org/QupZilla/qupzilla.svg?branch=master)](https://travis-ci.org/QupZilla/qupzilla)  
-Homepage: [http://www.qupzilla.com](http://www.qupzilla.com)  
-Blog: [http://blog.qupzilla.com](http://blog.qupzilla.com)  
-IRC: `#qupzilla` at `irc.freenode.net`  
+[![Build Status](https://travis-ci.org/QupZilla/qupzilla.svg?branch=master)](https://travis-ci.org/QupZilla/qupzilla)
+Homepage: [http://www.qupzilla.com](http://www.qupzilla.com)
+Blog: [http://blog.qupzilla.com](http://blog.qupzilla.com)
+IRC: `#qupzilla` at `irc.freenode.net`

```

Comment:
They might indeed... however my editor is permanently set to strip trailing spaces and is done automatically. The MD could use some fine tuning for sure outside of this issue.